### ISSUE TITLE
Finding Neo

### ISSUE THREAD

**Issue by cosmic-springs** [neo-project/neo-gui#199]
Hi.
I have now lost 9 NEO and around 2000 DBC. They disappeared from my wallet and the NEO are showing in my transaction history (-9). However, no transaction was made by me?? I accessed neotracker.io the previous day. 
It’s everything i had :( so disappointed. I realise that people have lost much more than me and I feel bad for them but it doesn’t make me feel better. it makes me so mad that people can get away with peoples hard earned money with what appears like very little effort. This was also a link that you assume is legit as it seems to be endorsed/embeded in the NEON wallet.
It would be great if rather than many different wallets we could access 1 or 2 that are solid and safe?
Unfortunately I am currently unemployed but used work with computers and i know that most issues users have are user error related and very frustrating constantly defended and justifying your work. The issues i have encountered are not simple user error. I cant help thinking that in order to catch many of the culprits of these scams/thefts wallet developers should look closer to home?? 
I’m not trying to get rich (9 NEO are unlikely to do that especially as I bought very high) I’m just trying to put a little by for when my kids get older.

Hash: 49774c05a3c7a7d9e90912f168e834077988f58f806441469f04bca0ffa3021c
Time: 07/02/2018, 17:17:19
Wallet: AU6GkGRt4jeESJnM56mz9uJiwacNMTHjzH

Have I been hacked/Phished/will I ever see my coins again?
Is this just part of the mysterious disappearing and often reappearing of NEO from various wallets and ledgers (I lost one around a month ago but it reappeared a week later)?
I may be a little naive but I’m not a complete idiot and I do take precautions to maximise security.

Any assistance/advice/HELP! Would be greatly appreciated.

**Comment by jrsikors** [comment]
See #188  .. sorry man :(

**Comment by erikzhang** [comment]
Duplicate of #198

**Comment by cosmic-springs** [comment]
So you close it? There are multiple victims of this theft going to the same address. I visited neotraker.io<http://neotraker.io> which was bookmarked and as far I can see endorsed by Neo. The phishing software was embedded into neotracker, it wasn’t a separate link. Neotracker are aware of this breach as far I can tell from snippets online. I followed correct security protocols the responsibility surely lies with Neo? If you were an exchange you would compensate.

Sent from my iPhone

On 26 Feb 2018, at 10:43 AM, Erik Zhang <notifications@github.com<mailto:notifications@github.com>> wrote:


Duplicate of #198<https://github.com/neo-project/neo-gui/issues/198>
